File: taotao-cloud-java/taotao-cloud-javase/src/main/java/com/taotao/cloud/java/javase/day01/Demo01/HelloWorld.java
package com.taotao.cloud.java.javase.day01.Demo01;

import java.util.Map;

public class HelloWorld {
	public static void main(String[] args) {
		System.out.print("Hello Everyone");
	}
}

class GoodByeWorld {
	public static void main(String[] args) {
		System.out.println("output environment var:");
		for(Map.Entry entry:System.getenv().entrySet()){
			System.out.println(entry.getKey()+":"+entry.getValue());
		}

	}


}

File: taotao-cloud-scala/src/main/scala/com/taotao/cloud/scala/atguigu/chapter01/HelloWorld.scala
package com.taotao.cloud.scala.atguigu.chapter01

/*
   object: 关键字，声明一个单例对象（伴生对象）
 */
object HelloWorld {
  /*
    main 方法：从外部可以直接调用执行的方法
    def 方法名称(参数名称: 参数类型): 返回值类型 = { 方法体 }
   */
  def main(args: Array[String]): Unit = {
    println("hello world")
    System.out.println("hello scala from java")
  }
}
